docs/Config.md

```diff
@@ -44,6 +44,7 @@ gui:
   showRandomTip: true
   showCommandLog: true
   commandLogSize: 8
+  sortTagsDescending: false # sort order for tags
```

Comment:
I actually think we shouldn't add a config for this: I think everybody would want tags to be ascending. If we find there's a demand for it, I think we should add the config option then. Better to err on the side of simplicity for now :)

Reply:
Well, I see, keep it simple is a good approach. But are you still willing to merge the change as a "hidden feature"? Without the config part?

Reply:
Yep! I consider it a mistake that it was ever sorted ascending because people typically care about recent tags

Reply:
Ok, I get it now, I think I was a bit confused by your first comment:
> "I think everybody would want tags to be ascending."

I'll change the PR later when I'm back home.